CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ if (CMAKE_BINARY_DIR STREQUAL CMAKE_SOURCE_DIR)
     message(FATAL_ERROR "Building in-source is not supported! Create a build dir and remove ${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 endif()
 
+project(openrct2)
```

Comment:
This cmake file includes other cmake files which set project. So I don't know if this is a good idea, can it be after the included cmake files?